docs/source/algorithms/ConvertToConstantL2-v1.rst

```diff
@@ -0,0 +1,32 @@
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+.. warning::
+
+   This algorithm will be deprecated in the next version of Mantid. Please, use :ref:`algm-ConverToConstantL2` instead, which
+   is the new name for this algorithm.
```

Comment:
Yes, it should! It also seems I had not set the deprecated algorithm quite correctly, the documentation should show up correctly now.

I have also added the usage example, but in doing so I am a bit suspicious a monitor is being moved that should not be, so I will mark this as in progress for now and take a look again next week.